packages/settings/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@ui5-language-assistant/settings",
+  "version": "0.0.1",
```

Comment:
should initial version 0.1.0 or 0.0.1?

Reply:
I think when we release it will increase the version to 1.0.0 or 0.1.0 (not sure which) anyway

Reply:
we will monitor it